playgrounds/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "packageManager": "yarn@3.6.0",
+  "private": true,
+  "workspaces": [
+    "*"
+  ],
+  "resolutions": {
+    "@slicemachine/adapter-next": "link:../packages/adapter-next",
```

Comment:
It doesn't seem like we can, at least not without other changes. Using `portal:` causes version mismatch issues:

```
➤ YN0000: ┌ Link step
➤ YN0071: │ Cannot link @slicemachine/manager into start-slicemachine@portal:../packages/start-slicemachine::locator=root-workspace-0b6124%40workspace%3A. dependency @prismicio/mocks@npm:2.0.0 conflicts with parent dependency @prismicio/mocks@npm:2.1.1
➤ YN0071: │ Cannot link @slicemachine/manager into slice-machine-ui@portal:../packages/slice-machine::locator=root-workspace-0b6124%40workspace%3A. dependency @prismicio/mocks@npm:2.0.0 conflicts with dependency @prismicio/mocks@npm:2.1.1 from sibling portal start-slicemachine
```

I'm not 100% sure what the cause is. I don't think keeping all of our dependencies on the same version is the right solution since playgrounds should be able to use any version.